[2016-12-19 15:16:08.067] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getAchievement (G:\ShouMedia\server\router\achievement\core.js:4:18)
       at G:\ShouMedia\server\router\achievement\getAchievement.js:13:43
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:16:28.005] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getAchievement (G:\ShouMedia\server\router\achievement\core.js:4:18)
       at G:\ShouMedia\server\router\achievement\getAchievement.js:13:43
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:16:30.501] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getAchievement (G:\ShouMedia\server\router\achievement\core.js:4:18)
       at G:\ShouMedia\server\router\achievement\getAchievement.js:13:43
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:18:25.744] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getAchievement (G:\ShouMedia\server\router\achievement\core.js:5:18)
       at G:\ShouMedia\server\router\achievement\getAchievement.js:13:43
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:23:53.838] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getAchievement (G:\ShouMedia\server\router\achievement\core.js:5:18)
       at G:\ShouMedia\server\router\achievement\getAchievement.js:13:43
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:32:33.612] [ERROR] log.js - { err: true }
[2016-12-19 15:38:41.105] [ERROR] log.js - { err: true }
[2016-12-19 15:38:58.344] [ERROR] log.js - { err: true }
[2016-12-19 15:39:15.119] [ERROR] log.js - { err: true }
[2016-12-19 15:41:37.429] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getBalance (G:\ShouMedia\server\lib\getCard.js:5:16)
       at G:\ShouMedia\server\router\getBalance.js:13:32
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:49:45.987] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getBalance (G:\ShouMedia\server\lib\getCard.js:5:16)
       at G:\ShouMedia\server\router\getBalance.js:13:32
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 15:51:18.820] [ERROR] log.js - { err: 
   Error: Invalid URI "/"
       at Request.init (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:274:31)
       at new Request (G:\ShouMedia\server\node_modules\.2.79.0@request\request.js:128:8)
       at request (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:54:10)
       at Function.get (G:\ShouMedia\server\node_modules\.2.79.0@request\index.js:62:12)
       at G:\ShouMedia\server\lib\get.js:5:17
       at Object.get (G:\ShouMedia\server\lib\get.js:4:12)
       at Object.getBalance (G:\ShouMedia\server\lib\getCard.js:5:16)
       at G:\ShouMedia\server\router\getBalance.js:13:32
       at process._tickCallback (internal/process/next_tick.js:103:7) }
[2016-12-19 17:17:04.942] [ERROR] log.js - TypeError: this.db.update is not a function
    at Object.module.exports (G:\ShouMedia\server\router\login\login.js:37:27)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2016-12-19 20:03:53.981] [ERROR] log.js - { Error: Cannot find module './../../lib/mongo'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (G:\ShouMedia\server\router\curriculum\core.js:3:8)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at G:\ShouMedia\server\router\curriculum\getClass.js:11:28
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2016-12-19 20:35:04.226] [ERROR] log.js - ReferenceError: name is not defined
    at Object.module.exports (G:\ShouMedia\server\router\address\get.js:6:64)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:69:7)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:69:7)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2016-12-20 00:23:09.841] [ERROR] log.js - TypeError: Cannot use 'in' operator to search for '_id' in A033-204-1
    at Document.$__buildDoc (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\document.js:206:27)
    at new Document (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\document.js:62:20)
    at Document.Subdocument (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\types\subdocument.js:15:12)
    at Document.Embedded._embedded (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\schema\embedded.js:27:17)
    at SchemaType.Embedded.castForQuery (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\schema\embedded.js:142:10)
    at Query._castUpdateVal (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2638:17)
    at Query._walkUpdatePath (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2582:25)
    at Query._castUpdate (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2454:23)
    at Query.update (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2231:22)
    at Function.update (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\model.js:2099:13)
    at G:\ShouMedia\server\router\infoPlus\getInfoPlus.js:12:43
[2016-12-20 00:23:40.982] [ERROR] log.js - TypeError: Cannot use 'in' operator to search for '_id' in A033-204-1
    at Document.$__buildDoc (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\document.js:206:27)
    at new Document (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\document.js:62:20)
    at Document.Subdocument (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\types\subdocument.js:15:12)
    at Document.Embedded._embedded (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\schema\embedded.js:27:17)
    at SchemaType.Embedded.castForQuery (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\schema\embedded.js:142:10)
    at Query._castUpdateVal (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2638:17)
    at Query._walkUpdatePath (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2582:25)
    at Query._castUpdate (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2454:23)
    at Query.update (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\query.js:2231:22)
    at Function.update (G:\ShouMedia\server\node_modules\.4.7.2@mongoose\lib\model.js:2099:13)
    at G:\ShouMedia\server\router\infoPlus\getInfoPlus.js:12:43
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 19:40:43.502] [ERROR] log.js - ReferenceError: info is not defined
    at valid (G:\ShouMedia\server\router\login\urpLogin.js:11:22)
    at Object.module.exports (G:\ShouMedia\server\router\login\urpLogin.js:9:27)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:69:7)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:69:7)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 19:51:16.916] [ERROR] log.js - ReferenceError: requie is not defined
    at Object.module.exports (G:\ShouMedia\server\router\achievement\getAchievement.js:6:23)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 19:51:30.430] [ERROR] log.js - { msg: '账号密码错误', pass: false }
[2017-02-16 19:51:54.521] [ERROR] log.js - { msg: '账号密码错误', pass: false }
[2017-02-16 20:10:20.079] [ERROR] log.js - SyntaxError: unmatched pseudo-class :eq
    at module.exports.compile [as pseudo] (G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\pseudos.js:388:10)
    at G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:99:26
    at Array.reduce (native)
    at compileRules (G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:97:15)
    at G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:87:32
    at Array.map (native)
    at compileToken (G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:87:4)
    at compileUnsafe (G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:32:9)
    at Function.compile (G:\ShouMedia\server\node_modules\.1.2.0@css-select\lib\compile.js:20:13)
    at Array.<anonymous> (G:\ShouMedia\server\node_modules\.0.22.0@cheerio\lib\api\traversing.js:317:23)
    at exports.children (G:\ShouMedia\server\node_modules\.0.22.0@cheerio\lib\api\traversing.js:288:25)
    at Object.<anonymous> (G:\ShouMedia\server\router\achievement\getAchievement.js:20:37)
    at exports.each (G:\ShouMedia\server\node_modules\.0.22.0@cheerio\lib\api\traversing.js:300:24)
    at Object.module.exports (G:\ShouMedia\server\router\achievement\getAchievement.js:17:39)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
[2017-02-16 20:10:33.366] [ERROR] log.js - TypeError: $(...).children(...)[0].text is not a function
    at Object.<anonymous> (G:\ShouMedia\server\router\achievement\getAchievement.js:20:55)
    at exports.each (G:\ShouMedia\server\node_modules\.0.22.0@cheerio\lib\api\traversing.js:300:24)
    at Object.module.exports (G:\ShouMedia\server\router\achievement\getAchievement.js:17:39)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 20:24:54.436] [ERROR] log.js - { Error: Cannot find module '../../lib/getUrpCookie'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at G:\ShouMedia\server\router\curriculum\core.js:6:16
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2017-02-16 20:25:03.089] [ERROR] log.js - { Error: Cannot find module '../../lib/getUrpCookie'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at G:\ShouMedia\server\router\curriculum\core.js:6:16
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2017-02-16 21:00:32.472] [ERROR] log.js - { Error: Cannot find module '../../lib/getUrpCookie'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at G:\ShouMedia\server\router\curriculum\core.js:6:16
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2017-02-16 21:01:07.727] [ERROR] log.js - { Error: Cannot find module './core'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at G:\ShouMedia\server\router\getCurriculum.js:43:28
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2017-02-16 21:01:30.974] [ERROR] log.js - { Error: Cannot find module './../login/getInfo'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.module.exports (G:\ShouMedia\server\router\getCurriculum.js:54:30)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:69:7)
    at G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:54:5
    at Object.co (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:50:10)
    at Object.toPromise (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:118:63)
    at next (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:99:29) code: 'MODULE_NOT_FOUND' }
[2017-02-16 21:01:47.138] [ERROR] log.js - { Error: Cannot find module './../../lib/getUrpCore'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at get (G:\ShouMedia\server\router\getCurriculum.js:6:12)
    at Object.module.exports (G:\ShouMedia\server\router\getCurriculum.js:55:29)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:103:7) code: 'MODULE_NOT_FOUND' }
[2017-02-16 21:02:35.877] [ERROR] log.js - ReferenceError: list is not defined
    at Object.module.exports (G:\ShouMedia\server\router\getCurriculum.js:63:21)
    at next (native)
    at onFulfilled (G:\ShouMedia\server\node_modules\.4.6.0@co\index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:27:59.082] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:42:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:29:11.462] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:42:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:32:11.700] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:43:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:32:29.202] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:43:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:32:38.521] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:43:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 21:34:46.152] [ERROR] log.js - TypeError: $(...).get(...).children is not a function
    at G:\ShouMedia\server\router\getCurriculum.js:8:27
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 22:00:34.426] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:41:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 22:00:55.819] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:41:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 22:01:01.918] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:41:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2017-02-16 22:01:59.472] [ERROR] log.js - TypeError: Cannot read property '0' of undefined
    at G:\ShouMedia\server\router\getCurriculum.js:41:37
    at process._tickCallback (internal/process/next_tick.js:103:7)
